src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -20,6 +20,26 @@ export default class DataUploader extends LightningElement {
         let textBlob: string = this.getValueFromInput('textArea');
         let dataList: GameData[] = this.processLine(textBlob);
         let errorMessages = validateInput(dataList);
+        //get file values
```

Comment:
https://betterprogramming.pub/handling-file-inputs-with-javascript-9f2d3a007f05
In connectedCallback, run something like:
```
const fileInput = document.getElementById('input');
fileInput.onchange = () => {
```